### ISSUE TITLE
Import resources fails on OpenShift 

### ISSUE THREAD

**Issue by Megan-Wright** [tektoncd/dashboard#1362]
# Expected behavior
 When accessing the dashboard using my OpenShift cluster I should be able to import resources successfully 

# Actual behavior
Import resources fails without even running the pipeline

<img width="1654" alt="Screenshot 2020-05-07 at 16 44 15" src="https://user-images.githubusercontent.com/52741262/81315289-0c9d3900-9082-11ea-9223-1ea4a22c1450.png">

# Steps to reproduce the problem

1. Try to import resources into an openshift cluster
2. Notice it failing without running the pipeline
3.

# Environment
- Kubernetes Platform:

OpenShift

- Tekton Pipelines version:

0.12.0

- Tekton Triggers version:

0.4.0

- Tekton Dashboard version:

master

# Additional Info

**Comment by eddycharly** [comment]
I changed a lot of things in the import resources process, i will have a look.
/assign

**Comment by eddycharly** [comment]
It looks like it's an issue with the service account:
```
User "system:serviceaccount:tekton-pipelines:default" cannot get resource "tasks" in API group "tekton.dev" in the namespace "bbb"
```

Did you use a service account for importing the resources ?

**Comment by eddycharly** [comment]
Tried again with pipelines `v0.11.0` and didn't face the issue.
Pipelines version : v0.11.0
Dashboard version : master
Repo : https://github.com/tektoncd/catalog
Directory: buildah
Service account: tekton-dashboard

**Comment by Megan-Wright** [comment]
> Tried again with pipelines `v0.11.0` and didn't face the issue.
> Pipelines version : v0.11.0
> Dashboard version : master
> Repo : https://github.com/tektoncd/catalog
> Directory: buildah
> Service account: tekton-dashboard

oh that's my bad completely, I thought I was on pipelines 0.11.0 but turns out I was on 0.12.0.. I've updated the issue now - you're right it's working fine for me on 0.11.0 as well

**Comment by eddycharly** [comment]
Can we close the issue or you need to check in more depth ?

**Comment by Megan-Wright** [comment]
This can be closed now, fixed as part of the work to get triggers 0.5.0 working with the dashboard.
/close
